### ISSUE TITLE
[question] New colors ?

### ISSUE THREAD

**Issue by gbdomubpkm** [wdehoog/shoutcast-ubports#16]
It's just a question: don't you think new colours should be added to the app ?
 
@cibersheep As Ubports colour specialist, don't you think? a New look for the app ?

Personally, I'll see maybe the app built on a blue base resting on the style of the Recorder app.

**Comment by cibersheep** [comment]
As much as we love the [Suru palette](https://ci.ubports.com/job/docs.ubports.com/job/PR-273/lastSuccessfulBuild/artifact/_build/html/humanguide/design-concepts/system-palette.html), that's only required for the core apps and system.

Developers should be able to choose :)

**Comment by mateosalta** [comment]
I would use the yellow you have in the logo in the header - black text on that.
For the titles, maybe the titles could be blue - they work for hyperlink like things. But perhaps a slightly larger font could work in black(theme color for dark theme compatibility) for those too.

maybe play with the grey color from the openstore for the bottom play bar

**Comment by wdehoog** [comment]
I tried to use suru colors. Something like this?
![Panorama-small](https://user-images.githubusercontent.com/3353303/87223515-f5c6db80-c37d-11ea-9278-a8ce80f4f521.jpg)

**Comment by mateosalta** [comment]
the text on the last two are better, for the main page maybe more like: 
![Screenshot_2020-07-11_08-59-44](https://user-images.githubusercontent.com/1965279/87225829-5261d080-c355-11ea-9e2a-4085472a66fe.png) (without the orange text on bottom)

I like the header divider too, if you think the yellow background is too much

**Comment by gbdomubpkm** [comment]
When this is possible, I think it would be easier to view the changes of all pages on a .click test (real situation).

**Comment by wdehoog** [comment]
@mateosalta How did you style the buttons and will that also work in dark mode? The suru icons are very thin and tend to 'disappear' on a colored background.

**Comment by cibersheep** [comment]
I agree that the header white icons disappear with the yellow background but you can set another color for the header elements :D or use a more dark yellow

I kinda like the yellow and grey blue you chose. Unspectedly nice

**Comment by mateosalta** [comment]
> @mateosalta How did you style the buttons and will that also work in dark mode? The suru icons are very thin and tend to 'disappear' on a colored background.

I had to use the system icons, they change color with theme (also if you change the icons to the same color as they system ones they will be themed I think). and then just used theme.palette.normal.background for the button background

**Comment by mateosalta** [comment]
playing around with diffrent stuff:
![Screenshot_2020-07-12_18-33-18](https://user-images.githubusercontent.com/1965279/87259091-3b62d180-c46e-11ea-81a5-36a406446829.png)
